### DISCUSSION TITLE
youtube-dl vs. yt-dlp

### DISCUSSION THREAD

**Discussion by ThomasSeeker** [smplayer-dev/smplayer#278] [General]
I just upgraded to Version: 21.10.0 (Revision 10000). 

Now youtube videos don't play anymore. It asks me to install yt-dlp. But after successfull installation, it still does not play. 

Reverting back to youtube-dl in settings/network solved the problem. 

Best greetings.

**Comment by smplayer-dev** [comment]
What operating system are you using?

**Comment by ThomasSeeker** [comment]
Linux Mint 18.3

**Reply by smplayer-dev** [reply]
I think Linux Mint 18.3 is based on Ubuntu 16.4. Unfortunately the python 3 provided by this system is too old to run yt-dlp (it requires at least python 3.6).

You can install smplayer using a snap or flatpak package. Those packages provide a newer version of python 3.

**Reply by ThomasSeeker** [reply]
Thank you for looking into that!

Well, I rather use youtube-dl until I upgrade my system. I hope youtube-dl stays available.

Thanks again!
